extensions/ql-vscode/src/telemetry.ts

```diff
@@ -168,6 +168,23 @@ export class TelemetryListener extends ConfigListener {
     );
   }
 
+  sendUIInteraction(name: string) {
+    if (!this.reporter) {
+      return;
+    }
+
+    const isCanary = (!!CANARY_FEATURES.getValue<boolean>()).toString();
```

Comment:
Minor: This is already implemented: https://github.com/github/vscode-codeql/blob/koesie10/bundle-extension/extensions/ql-vscode/src/config.ts#L443-L446